src/main/java/com/itschool/eventmanagment/controllers/UserController.java

```diff
@@ -18,7 +18,7 @@ public UserController(UserService userService) {
     }
 
     @GetMapping("api/users/{id}/events")
-    public ResponseEntity<List<EventDTO>> getUserEvents(@PathVariable String id) {
-        return ResponseEntity.ok(userService.getUsersEvents());
+    public ResponseEntity<List<EventDTO>> getUserEvents(@PathVariable Long id) {
```

Comment:
Thank you take into consideration my suggestion regarding naming

Reply:
Thanks